content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -108,6 +108,13 @@ Cost allocation divides host compute and other resource costs from your cloud pr
 
 Use the `allocated_resource` tag to visualize the spend resource associated with your costs at various levels, including the Kubernetes node, container orchestration host, storage volume, or entire cluster level.
 
+### Kubernetes tag extraction
+
+Only _tags_ from the direct resource, such as a pod, as well as the underlying nodes are added to cost metrics by default. To include labels as tags, annotations as tags, or tags from related resources such as namespaces view the detailed guide on [Kubernetes Tag Extraction][201].
```

Comment:
```suggestion
Only _tags_ from the direct resource, such as a pod, as well as the underlying nodes, are added to cost metrics by default. To include labels as tags, annotations as tags, or tags from related resources such as namespaces, see [Kubernetes Tag Extraction][201].
```

Reply:
How would you feel about moving this section to below the tabs? My only concern is that this heading being above the tabs makes me think that maybe the tabs are _all_ to do with Kubernetes tag extraction, and that doesn't seem accurate. Moving this to below that section might help prevent that misunderstanding from happening.

Reply:
That seems fine to me, would I just have to copy and paste it 3 times? (once per cloud)

Reply:
Just checking in on this placement question (I realized it was set as resolved, so maybe it wasn't showing up) - let me know if you'd like assistance with moving it 🙂

Reply:
Sorry about the delay. I think I fixed it, feel free to take another look

Reply:
No problem, thanks for taking another look! When I look at your preview link, the new cost allocation section is appearing in the tabs under the prereqs heading. But it doesn't seem like a prereq? I think it might make more sense underneath the tabs altogether, unless it's actually a prereq?

Reply:
I dont see that. Do you have a screenshot? I see it under `allocate costs`
<img width="980" alt="image" src="https://github.com/user-attachments/assets/b128de61-99d3-4971-bb4f-568128e25233" />

Reply:
Awesome! I think you might have to merge this, I am not sure I can. But it is ready when you are

Reply:
I think anyone should be able to merge it using `/merge`, but I went ahead and did it 🙂 Thanks again!!